doc/source/conf.py

```diff
@@ -101,20 +101,20 @@
         reldir = os.path.relpath(dirname, source_path)
         for fname in fnames:
             if os.path.splitext(fname)[-1] in (".rst", ".ipynb"):
-                fname = os.path.relpath(os.path.join(dirname, fname), source_path)
+                fname_new = os.path.relpath(os.path.join(dirname, fname), source_path)
```

Comment:
this is on the right track, but in general, unless there's a meaningful name to give, slight preference for prefixing with an underscore (`_fname`)